library/repo.go

```diff
@@ -584,6 +611,7 @@ func (r *Repo) String() string {
   Org: %s,
   Private: %t,
   Timeout: %d,
+	Counter: %d,
```

Comment:
Could we fix the spacing here so it's inline with the other fields?

Reply:
Oh weird, my editor rendered it fine. I'll take a look